package/MDAnalysis/analysis/polymer.py

```diff
@@ -268,7 +268,8 @@ def plot(self, ax=None):
         """
         import matplotlib.pyplot as plt
         if ax is None:
-            ax = plt.gca()
+            plt.figure()
+            ax = plt.axes()
```

Comment:
This works but just does what most people would probably do
```python
ax = plt.subplot(111)
```
My personal preference is to use `subplot()` here but maybe others have other opinions?